rmgpy/data/thermo.py

```diff
@@ -1461,13 +1461,17 @@ def correct_binding_energy(self, thermo, species, metal_to_scale_from=None, meta
             find_cp0_and_cpinf(species, thermo)
 
         # now edit the adsorptionThermo using LSR
-        comments = []
-        for element in 'CHON':
-            if normalized_bonds[element]:
-                change_in_binding_energy = delta_atomic_adsorption_energy[element].value_si * normalized_bonds[element]
-                thermo.H298.value_si += change_in_binding_energy
-                comments.append(f'{normalized_bonds[element]:.2f}{element}')
-        thermo.comment += " Binding energy corrected by LSR ({})".format('+'.join(comments))
+        if metal_to_scale_to != metal_to_scale_from:
```

Comment:
Don't think this commit is needed. See https://github.com/ReactionMechanismGenerator/RMG-Py/commit/1756a659d233404b933f90f0a77822247b21b78d